SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -149,8 +149,8 @@ void USpatialWorkerConnection::Connect(bool bInitAsClient, uint32 PlayInEditorID
 	const USpatialGDKSettings* SpatialGDKSettings = GetDefault<USpatialGDKSettings>();
 	if (SpatialGDKSettings->bUseDevelopmentAuthenticationFlow && bInitAsClient)
 	{
-		LocatorConfig.WorkerType = SpatialConstants::DefaultClientWorkerType.ToString();
-		LocatorConfig.UseExternalIp = true;
+		DevAuthConfig.WorkerType = SpatialConstants::DefaultClientWorkerType.ToString();
```

Comment:
can't highlight the correct line, but will line 221 work? I think that part still uses the LocatorConfig.

Reply:
Should this use the devauthtoken?

Reply:
I guess this would have worked actually but wouldn't have been "correct".